### ISSUE TITLE
Image description: a dialog instance is opened everytime NVDA+g is pressed, when the feature is disabled

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#19302]
### Steps to reproduce
1. Open NVDA with AI image description disabled
2. Press nvda+g multiple times

### Actual behavior
On every press in step 2 a new dialog instance is envoked.

### Expected behavior
Only one dialog should be shown, on subsequent presses NVDA should not do anything.

#### NVDA installed/portable/running from source
Installed

#### NVDA version
alpha-53667,bc2647d0 (2026.1.0.53667)

#### Windows version
Windows 11 25 H2

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors
n/a

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
cc: @tianzeshi-study, @gerald-hartig

**Comment by tianzeshi-study** [comment]
Yes, we should check whether the dialog has already been opened before showing it.

Alternatively, we could simply remove the dialog altogether — pressing the gesture could directly enable image descriptions for the current session and describe the current image.

**Comment by gerald-hartig** [comment]
@Adriani90 there's no need to @ NV Access staff on newly created issues. They will get reviewed during our weekly triage.

**Comment by seanbudd** [comment]
@tianzeshi-study - yes, please check if the dialog is open before opening a new one.
Can we remove the dialog? Don't we need to prompt a download of the model in some cases? I agree that a dialog is unnecessary except for the model download
